csuntechlab/iSTART
# STR-250 -- FE: Refactor informational template

# Description
Decoupled shared module templates and re-organized file/folder structure.

# Testing
1. Check to see if all previous functionality for templates works.
2. Check to ensure all video templates are properly placed.
3. Ensure all slide template folder/file structure makes sense.
